src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/GenomicsDBProgramGroup.java

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+
+public class GenomicsDBProgramGroup implements CommandLineProgramGroup {
```

Comment:
Delete this new program group, and make your tool part of the existing `VariantProgramGroup`